api/v1alpha1/hardwareclassification_types.go

```diff
@@ -60,12 +60,14 @@ type Cpu struct {
 	// MinimumSpeed of cpu should be greater than 0
 	// Ex. MinimumSpeed > 0
 	// Ex. MinimumSpeed: 2600
+	// User wants CPU speed 2.6 (in GHz), then s/he should specify as 2600 MHz
```

Comment:
The field name really should include the units so someone doesn't have to look it up separately.

Reply:
okay.

Reply:
Changed existing field to MaximumSpeedMHz & MinimumSpeedMHz which.